cmd/geth/main.go

```diff
@@ -151,6 +151,7 @@ var (
 		utils.GpoMaxGasPriceFlag,
 		utils.GpoIgnoreGasPriceFlag,
 		utils.MinerNotifyFullFlag,
+		utils.IgnoreLegacyReceiptsFlag,
```

Comment:
You should also add it to flag helper https://github.com/ethereum/go-ethereum/blob/master/cmd/geth/usage.go#L32